rclcpp/include/rclcpp/node.hpp

```diff
@@ -864,18 +869,40 @@ class Node : public std::enable_shared_from_this<Node>
   std::vector<std::string>
   get_node_names() const;
 
+  /// Return a map of existing topic names to list of topic types.
+  /**
+   * \return a map of existing topic names to list of topic types.
+   * \throws std::runtime_error anything that rcl_error can throw
+   */
   RCLCPP_PUBLIC
   std::map<std::string, std::vector<std::string>>
   get_topic_names_and_types() const;
 
+  /// Return a map of existing service names to list of service types.
+  /**
+   * \return a map of existing service names to list of service types.
+   * \throws std::runtime_error anything that rcl_error can throw
+   */
   RCLCPP_PUBLIC
   std::map<std::string, std::vector<std::string>>
   get_service_names_and_types() const;
```

Comment:
This doc string looks like a copy-paste error. There were some additional fixes for it, but the brief is still wrong.

Also, something odd is happening where the doc string for the next method, `count_publishers()`, is not rendering:

https://docs.ros2.org/foxy/api/rclcpp/classrclcpp_1_1Node.html#aa3df8a7d8b174f8049dfd2766f4b2eb9

Reply:
https://github.com/ros2/rclcpp/pull/1719